lib/layout/dummyLayoutV1.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * dummyLayoutV1.js
+ *
+ * this dumps a fake graph
+ *
+ * @author John O'Brien. 
+ */
+
+// Create a graph instance.
+
+
+function buildGraph() {
+  var lib = './../';
+  var Graph = require(lib + 'model/graph');
+  var Node = require(lib + 'model/node');
+  var Port = require(lib + 'model/port');
+  var CardinalPortPosition = require(lib + 'model/cardinalPortPosition');
+  var nodes = [];
+  //loop for building the nodes
+  for(var i =  0; i < 4; i++){
+    //creates the ports
+    var portIn = new Port( ( i * 100 ) + 01, 'input');
+    if( i === 3 ){
```

Comment:
Then name it accordingly. It's generally nicest if the only numerical literals occur on the left sides of assignment statements. It helps people reading the code to understand what 3 represents here and/or why you chose that value.